packages/events/src/EventBoundary.ts

```diff
@@ -977,6 +977,13 @@ export class EventBoundary
             if (clickEvent.pointerType === 'mouse')
             {
                 this.dispatchEvent(clickEvent, 'click');
+
+                const isRightButton = clickEvent.button === 2;
+
+                if (isRightButton)
+                {
+                    this.dispatchEvent(clickEvent, 'rightclick');
+                }
```

Comment:
I thought about it too. But I noticed that pointer events are also triggered by right click like in issue playground:

`trigger right click`

Output:
`onpointerdown`
`onrightdown`
`onpointerup`
`onrightup`
`onclick`

Reply:
But on second thought, it's best to just let one event fire, Thanks.

I'll correct the length of the line.

Reply:
Let's see what others think about this. You can fix the lint problem first.

Reply:
I agree that it's better to be consistent with rightdown/mousedown vs pointerdown. So I would be in favor of firing one event.

Long-term, I don't like the `right*` events and handlers. They are not part of the DOM spec, and I'm not sure how much convenience they add since it's pretty easy to check for `button` property.

Reply:
It seems that it is called `auxclick` in DOM :thinking: